aula-pratica-02/ex7.c

```diff
@@ -0,0 +1,31 @@
+#include<stdio.h>
+int ler();
+int mult(int a, int b);
+int imprimir(int a);
+
+int ler()
+{
+    int a, b;
+    printf("Primeiro numero: ");
+    scanf("%d", &a);
+    printf("Segundo numero: ");
+    scanf("%d", &b);
+    return mult(a, b);
+}
+
+int mult(int a, int b)
+{
+    return imprimir(a * b);
+}
+
+int imprimir(int a)

```

Comment:
as the 1 is not doing anything really, you can tell the C compiler that it returns nothing
```suggestion
void imprimir(int a)
```